static/app/views/explore/contexts/logs/logsPageData.tsx

```diff
@@ -37,9 +38,6 @@ export function LogsPageDataProvider({children}: {children: React.ReactNode}) {
 }
 
 export function useLogsPageDataQueryResult() {
-  const hasInfiniteFeature = useOrganization().features.includes(
-    'ourlogs-infinite-scroll'
-  );
   const pageData = useLogsPageData();
-  return hasInfiniteFeature ? pageData.infiniteLogsQueryResult : pageData.logsQueryResult;
+  return pageData.infiniteLogsQueryResult;
```

Comment:
Can we delete `logsQueryResult` from the context now?

Reply:
Already was doing it